build_all/base-libraries/AzureIoTUtility/src/esp8266/sample_init.cpp

```diff
@@ -4,10 +4,13 @@
 #ifdef ARDUINO_ARCH_ESP8266
 
 #include <Arduino.h>
+#undef round
 #include <time.h>
 #include <ESP8266WiFi.h>
-#include <WiFiClientSecure.h>
+// #include <WiFiClientSecure.h>
```

Comment:
if this isn't needed at all it should probably be removed

Reply:
Done.